config/routes.rb

```diff
@@ -11,5 +11,5 @@
   root to: 'fee_categories#index'
 
   get 'search/fees'
-  get 'fee_results/:query', to: 'search#fee_results', as: 'fee_results'
+  get 'fee_results', to: 'search#fee_results', as: 'fee_results'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.